ui/app/selectors/selectors.js

```diff
@@ -353,6 +388,34 @@ function getCustomNonceValue (state) {
   return String(state.metamask.customNonceValue)
 }
 
+function getAllPermissions (state) {
+  return state.metamask.domains || {}
```

Comment:
I'm having trouble locating this state in the background 🤔

Reply:
but also, this function is not used anywhere

Reply:
removed in 92e78ad58

Reply:
That `domains` state is used in `getPermissionsDomains` and `getAddressConnectedDomainMap` as well

Reply:
@Gudahtt `domains` is added to state and controlled by rpc-cap: see https://github.com/MetaMask/rpc-cap/blob/master/index.ts or `metamask-extension/node_modules/rpc-cap/dist/index.js`